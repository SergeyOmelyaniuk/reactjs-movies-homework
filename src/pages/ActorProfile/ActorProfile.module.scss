.actorProfile {
	padding-top: 32px;
	@media (max-width: 1600px) {
		padding-top: 26px;
	}
}

.titleKnownBy {
	font-size: 34px;
	text-transform: uppercase;
	display: block;
	margin-bottom: 50px;
	@media (max-width: 1600px) {
		font-size: 28px;
		margin-bottom: 41px;
	}
}
