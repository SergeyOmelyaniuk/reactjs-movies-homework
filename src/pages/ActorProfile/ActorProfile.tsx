import React from 'react';
import ActorDescription from '../../сomponents/ActorDescription';
import WrapMovies from '../../сomponents/WrapMovies';
import styles from './ActorProfile.module.scss';
import { movies } from '../../mocks/movies';

function ActorProfile() {
	return (
		<div className={styles.actorProfile}>
			<ActorDescription actor={movies.popular[0].actors[0]} />
			<div className={styles.knownBy}>
				<span className={styles.titleKnownBy}>Known by</span>
				<WrapMovies movies={movies.top_rated} />
			</div>
		</div>
	);
}

export default ActorProfile;
