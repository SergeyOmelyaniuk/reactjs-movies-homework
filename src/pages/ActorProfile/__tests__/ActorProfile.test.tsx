import React from 'react';
import { render } from '@testing-library/react';
import ActorProfile from '..';

describe('ActorProfile', () => {
	test('should render', () => {
		const { container } = render(<ActorProfile />);

		expect(container).toMatchSnapshot();
	});
});
