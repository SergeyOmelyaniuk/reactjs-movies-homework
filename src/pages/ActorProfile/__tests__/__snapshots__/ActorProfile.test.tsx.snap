// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ActorProfile should render 1`] = `
<div>
  <div
    class="actorProfile"
  >
    <div
      class="actorDescription"
    >
      <div
        class="wrapPhotoActor"
      >
        <img
          alt=""
          class="photoActor"
        />
      </div>
      <div
        class="infoActor"
      >
        <h2
          class="nameActor"
        >
          Kate Beckinsale
        </h2>
        <div
          class="textBlock"
        >
          <span
            class="title"
          >
            Birthday
            :
          </span>
          <p
            class="text"
          >
            1959-04-15
          </p>
        </div>
        <div
          class="textBlock"
        >
          <span
            class="title"
          >
            Place of birth
            :
          </span>
          <p
            class="text"
          >
            Brest, Belarus
          </p>
        </div>
        <div
          class="textBlock"
        >
          <span
            class="title"
          >
            Biography
            :
          </span>
          <p
            class="text"
          >
            Text
          </p>
        </div>
        <div
          class="photosActor"
        >
          <span
            class="titlePhotosActor"
          >
            Photos
          </span>
          <div
            class="wrapPhotosActor"
          >
            <div
              class="wrapPhotoSmallActor"
            >
              <img
                alt=""
                class="photoSmallActor"
              />
            </div>
            <div
              class="wrapPhotoSmallActor"
            >
              <img
                alt=""
                class="photoSmallActor"
              />
            </div>
            <div
              class="wrapPhotoSmallActor"
            >
              <img
                alt=""
                class="photoSmallActor"
              />
            </div>
            <div
              class="wrapPhotoSmallActor"
            >
              <img
                alt=""
                class="photoSmallActor"
              />
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="knownBy"
    >
      <span
        class="titleKnownBy"
      >
        Known by
      </span>
      <div
        class="wrapMovies"
      >
        <div
          class="wrapMovie"
        >
          <div
            class="movieCard"
          >
            <div
              class="wrapImage"
            >
              <div
                class="overlay"
              />
              <img
                alt=""
                class="image"
                src="https://image.tmdb.org/t/p/w500/film_1.jpg"
              />
              <span
                class="rating"
              >
                68
              </span>
              <div
                class="preview"
              />
            </div>
            <div
              class="content"
            >
              <h2
                class="title"
              >
                Красотка на взводе
              </h2>
              <p
                class="genre"
              />
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
